packages/LUISGen/readme.md

```diff
@@ -52,7 +69,7 @@ The callback value app will be a strongly typed LUIS result.
 
 Nightly builds are based on the latest development code which means they may or may not be stable and probably won't be documented. These builds are better suited for more experienced users and developers although everyone is welcome to give them a shot and provide feedback.
 
-You can get the latest nightly build of MSBot from the [BotBuilder MyGet](https://botbuilder.myget.org/gallery) feed. To install the nightly - 
+You can get the latest nightly build of LUISGen from the [BotBuilder MyGet](https://botbuilder.myget.org/gallery) feed. To install the nightly - 
```

Comment:
After Ignite, all of our CLI packages will be published to npmjs registry so not sure if the comment need to be there?